samples/bluetooth/iso_connected_benchmark/src/main.c

```diff
@@ -146,7 +146,7 @@ static void print_stats(char *name, struct iso_recv_stats *stats)
 
 	LOG_INF("%s: Received %u/%u (%.2f%%) - Total packets lost %u",
 		name, stats->iso_recv_count, total_packets,
-		(float)stats->iso_recv_count * 100 / total_packets,
+		(double)((float)stats->iso_recv_count * 100 / total_packets),
```

Comment:
Can you elaborate on the change here and in the other ISO sample? 

Why is this being cast to a double? Is it because the `%.2f` format takes a double? 

In that case I would suggest replacing the cast to `float` to `double` as
```suggestion
		((double)stats->iso_recv_count * 100 / total_packets),
```

instead

Reply:
All `%f` with prints become doubles in this case.

I'd rather not do that. If I make it a double, it would do that entire math as a double, which could be a bit more expensive depending on the cpu configuration.